pkg/cmd/server/apis/config/v1/conversions.go

```diff
@@ -136,6 +136,13 @@ func SetDefaults_MasterConfig(obj *MasterConfig) {
 			obj.AdmissionConfig.PluginConfig[pluginName] = &AdmissionPluginConfig{}
 		}
 	}
+
+	// Set defaults Cache TTLs for external Webhook Token Reviewers
+	for _, wta := range obj.AuthConfig.WebhookTokenAuthenticators {
+		if len(wta.CacheTTL) == 0 {
+			wta.CacheTTL = "10s"
```

Comment:
@openshift/api-review any opinion on the default?

Reply:
Also, is not this mutating a copy instead of the real object?

Reply:
Should be fixed now, also changed default to 2m as per upstream